.travis.yml

```diff
@@ -9,6 +9,15 @@ os:
 
 sudo: false
 
+branches:
+  only:
+    # This is where pull requests from "bors r+" are built.
+    - staging
+    # This is where pull requests from "bors try" are built.
+    - trying
+    # Also build pull requests.
+    - master
```

Comment:
As long as this is activated PRs will still get checked by appveyor and travis (once).